.github/workflows/app-build-docker-image.yml

```
@@ -46,25 +54,41 @@ jobs:
     name: Determine Image Name
     runs-on: ubuntu-latest
     outputs:
-      docker-image-name-with-tag: "${{ env.DOCKER_IMAGE_NAME_WITH_TAG }}"
+      gchr-docker-image-name-with-tag: "${{ env.GCHR_DOCKER_IMAGE_NAME_WITH_TAG }}"
+      dockerhub-image-name-with-tag: "${{ env.DOCKERHUB_IMAGE_NAME_WITH_TAG }}"
```

There is a proposal below for a way to only get the image-name-with-tag once and then add the prepend at the point where these are used. An alternative would be to still get the image-name-with-tag only once and do the prepend here instead of at the point the tags are used.

```suggestion
      ghcr-docker-image-name-with-tag: "ghcr.io/${{ env.DOCKER_IMAGE_NAME_WITH_TAG }}"
      dockerhub-image-name-with-tag: "${{ env.DOCKER_IMAGE_NAME_WITH_TAG }}"
```